matrix_benchmarking/store/simple.py

```diff
@@ -189,10 +189,13 @@ def has_lts_anchor(files):
         pass
 # ---
 
-def parse_data(results_dir=None):
+def parse_data(results_dir=None, lts=False):
     if results_dir is None:
         results_dir = pathlib.Path(cli_args.kwargs["results_dirname"])
 
+    if lts:
+        return parse_lts_data(results_dir)
+
```

Comment:
this part, I need to work on it tomorrow, I don't like this idea (that I had) of adding a parameter, it breaks some backwards compatibility. I'll sort it out tomorrow